src/client/components/AddResume/AddResume.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import ReactDom from 'react-dom';
+import Button from '../Button/Button';
+import './AddResume.css';
+import close from '../../assets/images/closeIcon.svg';
+
+export const AddResume = () => {
+const[title,setTitle] = useState('');
+const[description,setDescription] = useState('');
+const[uploadFile,setUploadaFile] =useState('');
+
+    return ReactDom.createPortal ((
+        <div className = "popup">
+            <div className = "popup-form">
+                <img className="closeicon" src= {close} alt='closeIcon'/>
+                <h3>Upload new CV</h3>
+            <div className='form'>
+                <div className = "title-box">
+                    <label>Add Title</label><br/>
+                    <input type='text' value={title} onChange={(e)=> setTitle(e.target.value)}/><br/>
+                </div>
+                <div className = "description-box">
```

Comment:
I dont know how much information is in the box name!?

```suggestion
                <div className="description">
```